src/app/core/services/label.service.ts

```diff
@@ -55,22 +55,32 @@ export class LabelService {
   private static responseLabels: Label[] = Object.values(REQUIRED_LABELS.response);
   private static statusLabels: Label[] = Object.values(REQUIRED_LABELS.status);
   private static otherLabels: Label[] = Object.values(REQUIRED_LABELS.others);
-  private static labelArrays = {
+  private static allLabelArrays = {
     severity: LabelService.severityLabels,
     type: LabelService.typeLabels,
     response: LabelService.responseLabels,
     status: LabelService.statusLabels,
     others: LabelService.otherLabels
   };
+  private static testerLabelArrays = {
+    severity: LabelService.severityLabels,
+    type: LabelService.typeLabels,
+  };
 
   constructor(private githubService: GithubService) {
   }
 
-  public static getRequiredLabelsAsArray(): Label[] {
+  public static getRequiredLabelsAsArray(needAllLabels: boolean): Label[] {
     let requiredLabels: Label[] = [];
 
-    for (const category of Object.keys(this.labelArrays)) {
-      requiredLabels = requiredLabels.concat(this.labelArrays[category]);
+    if (needAllLabels) {
+      for (const category of Object.keys(this.allLabelArrays)) {
+        requiredLabels = requiredLabels.concat(this.allLabelArrays[category]);
+      }
+    } else {
+      for (const category of Object.keys(this.testerLabelArrays)) {
+        requiredLabels = requiredLabels.concat(this.testerLabelArrays[category]);
+      }
```

Comment:
A little bit subjective, but I'm thinking if this would make a cleaner style of code?

```
const labels = needLabels ? Object.values(allLabelArrays) : Object.values(testerLabelArrays);

labels.map(label => requiredLabels.concat(label));
```

It seems like we can remove the for loop too, because we are only concatenating the labels object (testerLabelArrays / alllabelArrays) values. What do you guys think @ducbinh2611 @CATcher-org/2021-devs

Reply:
I think this is a good suggestion!

Reply:
Thanks for the suggestion! I think it is definitely neater